api/api.proto

```diff
@@ -2729,6 +2744,53 @@ message SdkVolumeDetachRequest {
 message SdkVolumeDetachResponse {
 }
 
+// Defines a request to setup an ephemeral volume
+message SdkVolumeSetupEphemeralRequest {
```

Comment:
This is essentially all parameters required to perform Create, Attach, and Mount.

I modelled TeardownEphemeralRequest the same way.